pkg/worker/redis.go

```diff
@@ -219,6 +220,6 @@ func (w *RedisWorker) Stats(ctx context.Context) (Stats, error) {
 
 	return Stats{
 		EnqueuedJobs: uint64(count),
-		InFlight:     w.inFlight,
+		InFlight:     atomic.LoadInt64(&w.inFlight),
```

Comment:
This is the bugfix, the rest is just renaming of variables and setting up PoolSize to more reasonable number.

Reply:
I thought you actually need to define it as atomic... ILT